doc/rados/operations/pg-repair.rst

```diff
@@ -55,9 +55,9 @@ The "pg repair" command attempts to fix inconsistencies of various kinds. If "pg
 
 "pg repair" will not solve every problem. Ceph does not automatically repair placement groups when inconsistencies are found in them.
 
-The checksum of an object or an omap is not always available. If an object is updated while a scrub is underway, its checksum is marked invalid. Even though an authoritative copy was available when the scrub was underway, that copy's authoritativeness was questionable. Ceph therefore refuses to repair it even if the user runs "pg repair". This is one of the situations in which manual intervention is necessary.
+The checksum of an object or an omap is not always available. Checksums are calculated incrementally. If an object is updated in the middle of a scrub, the write operation involved in the update changes the object and invalidates its checksum. The whole object is not read while recalculating the checksum. So if an object has been updated under these conditions, Ceph doesn't repair it even if the user runs "pg repair". This is one of the situations in which manual intervention is necessary.
 
-The matched-record checksum and the calculated checksum cannot prove that the authoritative copy is in fact authoritative. It is possible that bitrot has changed the data on the disk in a way that makes the two digests match. This is why human intervention is necessary when an inconsistency is discovered.
+The material in this paragraph is relevant for filestore, and bluestore has its own internal checksums. The matched-record checksum and the calculated checksum cannot prove that the authoritative copy is in fact authoritative. It is possible that bitrot has changed the data on the disk in a way that makes the two digests match. This is why human intervention is necessary when an inconsistency is discovered.
```

Comment:
@zdover23 While this sentence is true "It is possible that bitrot has changed the data..." it is astronomically unlikely.  I might say this instead: "In the case that there is no checksum available "pg repair" favors the data on the primary.  This may or might not be the uncorrupted replica. This is why human intervention is sometimes necessary..."

Reply:
This change has been added to the documentation.